hack/third-party/inotify/inotify_stubs.c

```diff
@@ -28,7 +28,9 @@
 #include <caml/callback.h>
 #include <caml/unixsupport.h>
 
+#ifdef __linux__
 #include <features.h>
```

Comment:
This looks like dead code. Looking at https://github.com/facebook/hhvm/commit/d50dd15bc34c60a4bcc02dc6bd95fdf32effc54b (from ages ago), it was doing something like

```
#if __GLIBC__ >= 2 && __GLIBC_MINOR__ >= 4
```

so I'll delete this include when merging.